doc/api/worker_threads.md

```diff
@@ -91,6 +120,16 @@ added: v10.5.0
 An arbitrary JavaScript value that contains a clone of the data passed
```

Comment:
It might be good to include more of a description of what "clone" here means in general. I know there are a few bits and pieces of an explanation through the doc, but coalescing those into a single section with some information on what types of values *cannot* be cloned, would be good.

Reply:
@jasnell How does 919fd07151495f55c60c59b79862962d459e1665 sound to you?